scripts/exgdas_enkf_ecen.sh

```diff
@@ -360,7 +368,9 @@ done # loop over analysis times in window
 ################################################################################
 # Postprocessing
 cd $pwd
-[[ $mkdata = "YES" ]] && rm -rf $DATA
+if [[ "${mkdata}" == "YES" ]]; then
+    rm -rf "${DATA}"
+fi
 
 
-exit ${err}
+exit "${err}"
```

Comment:
Should all of the `err` instances become `status` in these scripts? Can be a follow-up PR, I need to review all `rm`s for a bugzilla anyway.

Reply:
I don't think so.  `err` needs to be exported when running `err_chk`, so it should not be replaced with `status`.  I think the last `exit` should simply be `exit 0` as all of the `export err=$?` calls are followed immediately by `err_chk`, so this should exit if `err` does not equal `0`.

Reply:
Sounds like we should replace all `status` with `err`. We have to move to use `err_chk` more anyway. Thoughts?

Reply:
That's probably a good idea.  I'll create an issue to handle both replacing `status` with `err` and using `err_chk` where appropriate.

Reply:
Opened https://github.com/NOAA-EMC/global-workflow/issues/3367.

Reply:
Thanks @DavidHuber-NOAA !